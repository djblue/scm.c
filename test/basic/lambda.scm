(write ((lambda (x) x) 0))
(write ((lambda (x y) (cons x y)) 1 2))
(write ((lambda x x) 1 2 3 4))
(write ((lambda (x . y) (cons x y)) 1 2 3 4))
(write ((lambda (x . y) (cons y x)) 1 2 3 4))
(write (procedure? if))
(write (procedure? +))
(write (procedure? (lambda (x) x)))
(write (procedure? #t))
(write (procedure? 1))
(write (procedure? "hello"))
