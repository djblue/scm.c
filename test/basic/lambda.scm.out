0
#t
#t
#t
#f
#f
#f
